ClientPlugin/Settings/Elements/Keybind.cs

```diff
@@ -0,0 +1,110 @@
+﻿using Sandbox.Game.Gui;
+using Sandbox.Graphics.GUI;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+using VRage.Game;
+using VRage.Input;
+using VRage.Utils;
+using VRage;
+using VRageMath;
+
+namespace ClientPlugin.Settings.Elements
+{
+    internal class KeybindAttribute : Attribute, IElement
+    {
+        public readonly string Description;
+
+        public Func<MyKeys> PropertyGetter;
+        public Action<MyKeys> PropertySetter;
+
+        public KeybindAttribute(string description = null)
+        {
+            Description = description;
+        }
+
+        public List<MyGuiControlBase> GetElements(string name, Func<object> propertyGetter, Action<object> propertySetter)
+        {
+            PropertyGetter = ()=>(MyKeys)propertyGetter();
+            PropertySetter = (MyKeys key)=>propertySetter(key);
+
+            var label = new MyGuiControlLabel(text: name);
+
+            var control = new MyControl(
```

Comment:
I could not figure out how to remove the keybind, e.g. assign no key.

Reply:
You can now unbind a key by right-clicking